src/catalog/search-configuration.md

```diff
@@ -61,17 +61,20 @@ With ElasticSearch, there is no out-of-the-box support for search by the suffix.
 
    Restricting this amount increases performance of searches and reduces the displayed list size. The default value is `8`.
 
-### Step 2: Configure the Elasticsearch connection
+### Step 2: Configure the Elasticsearch/OpenSearch connection
 
 {:.bs-callout-info}
 The **Search Engine**, **Elasticsearch Server Hostname**, **Elasticsearch Server Port**, **Elasticsearch Index Prefix**, **Enable Elasticsearch HTTP Auth**, and **Elasticsearch Server Timeout** fields were configured when Commerce was installed or upgraded. These values should only be changed when upgrading or modifying Elasticsearch.
 
+{:.bs-callout-info}
+For Magento 2.4.6 and higher you can set up OpenSearch instead of Elasticsearch. You need to choose the value `OpenSearch` for **Search Engine** setting.
```

Comment:
```suggestion
For Adobe Commerce and Magento Open Source 2.4.6 and higher, you can set up OpenSearch instead of Elasticsearch. To use OpenSearch, choose the `OpenSearch` value for the **Search Engine** setting.
```

Reply:
done